tool/grind.dart

```diff
@@ -91,6 +91,42 @@ serveLocalBackend() async {
   });
 }
 
+@Task('Serve locally on port 8000 and use beta server URL for pre null-safe')
+@Depends(ConstTaskInvocation(
+  'build',
+  ConstTaskArgs('build', flags: {
+    _debugFlag: true,
```

Comment:
taking this line out fixes the issue for me.

Reply:
Thanks @johnpryan, that fixed it nicely.

Reply:
I think this is because DDC doesn't use the same environment variables as dart2js, but @srawlins might know more.

Dropping this line will switch back to dartj2s, which makes sense for these tasks since they're for testing with a different backend, not making changes to the UI